src/scss/common/accessibility.scss

```
@@ -0,0 +1,38 @@
+$reduced-motion: false;
+
+@mixin not-reduced-motion {
+  @if not $reduced-motion {
+    @content;
+  }
+}
+
+@mixin reduced-motion-compliant-animation {
+  @content;
+  
+  // https://www.a11yproject.com/posts/understanding-vestibular-disorders/#what-should-you-consider%3F
+  @if $reduced-motion {
+    transition-duration: 0.001s !important;
+    animation-duration: 0.001s !important;
+  }
+}
+
+@mixin -reduced-motion {
+  $reduced-motion: true;
```

Seemingly setting a sass variable like this doesn't override the global version, only creates its own local copy - so this doesn't affect `not-reduced-motion` or `reduced-motion-compliant-animation`. It _does_ however work if we add a global flag to this:

```suggestion
  $reduced-motion: true !global;
```